tests/test_application.py

```diff
@@ -597,3 +597,33 @@ async def test_move_network_to_channel(app):
 
     assert len(app._api._queued_at.mock_calls) == 1
     app._api._queued_at.assert_any_call("SC", 1 << (26 - 11))
+
+
+async def test_energy_scan(app):
+    rssi = b"\x0A\x0F\x14\x19\x1E\x23\x28\x2D\x32\x37\x3C\x41\x46\x4B\x50\x55"
+    app._api._at_command = mock.AsyncMock(spec=XBee._at_command, return_value=rssi)
+    time_s = 3
+    energy = await app.energy_scan(
+        channels=[x for x in range(11, 27)], duration_exp=time_s, count=3
+    )
+    assert app._api._at_command.call_count == 3
+    assert app._api._at_command.call_args_list[0][0][0] == "ED"
```

Comment:
For readability, you can use `.mock_calls == [call("ED", time_s)]` after importing `call` from `unittest.mock`.

Reply:
Done